packages/account/src/Components/account-limits/account-limits-context.tsx

```diff
@@ -3,10 +3,13 @@ import * as React from 'react';
 export type TAccountLimitsContext = {
     currency: string;
     footer_ref?: React.RefObject<HTMLElement>;
-    overlay_ref?: React.RefObject<HTMLElement>;
-    toggleOverlay?: () => boolean;
+    overlay_ref: HTMLDivElement;
```

Comment:
the property name doesn't match with property value

Reply:
maybe `overlay_element` or `overlay_content`?
cc: @suisin-deriv, @yauheni-deriv

Reply:
I guess we don't need to change this as well :thinking: